test/index.js

```diff
@@ -100,33 +114,46 @@ describe('KeyringController', () => {
 
       const keyring = await keyringController.restoreKeyring(mockSerialized)
       assert.equal(keyring.wallets.length, 1, 'one wallet restored')
+
       const accounts = await keyring.getAccounts()
       assert.equal(accounts[0], addresses[0])
     })
   })
 
-  describe('#getAccounts', () => {
-    it('returns the result of getAccounts for each keyring', async () => {
+  describe('#getAccounts', function () {
+
+    it('returns the result of getAccounts for each keyring', async function () {
       keyringController.keyrings = [
-        { async getAccounts () { return [1, 2, 3] } },
-        { async getAccounts () { return [4, 5, 6] } },
+        {
+          getAccounts () {
+            return Promise.resolve([1, 2, 3])
+          },
+        },
+        {
+          getAccounts () {
+            return Promise.resolve([4, 5, 6])
+          },
+        },
       ]
 
       const result = await keyringController.getAccounts()
-      assert.deepEqual(result, [1, 2, 3, 4, 5, 6])
+      assert.deepEqual(result, ['0x01', '0x02', '0x03', '0x04', '0x05', '0x06'])
```

Comment:
Huh - why did this change? 🤔

Reply:
Because I changed `assert` to `strict`, and that's what is actually returned 😬